### PR TITLE
notes and questions n27460 - Add importmempool RPC

### PR DESCRIPTION
**Pull request by LarryRuane**
@jonatack @stickies-v

### CODE REVIEW

**Comment by LarryRuane** [comment]
Thanks, @jonatack, I force-pushed your suggestions.

File: _posts/2023-04-26-#27460.md [hunk 1]
```diff
@@ -7,21 +7,93 @@ authors: [MarcoFalke]
...
+  implemented persisting the mempool to disk so that its entries are
+  available after a restart. This PR was merged in
+  [v0.14.0](https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-0.14.0.md#retaining-the-mempool-across-restarts).
+
+- The `mempool.dat` file, located in the datadir, is a binary file in a
+  proprietary format, so it's not possible
+  to manually edit this file.
+
+- The in-memory mempool is not just a cache;
+  it contains the entire mempool rather than a subset (such as most-recently-used).
```
Comment (stickies-v) [inline _posts/2023-04-26-#27460.md:31]: Would slightly rephrase, e.g. like this?
```suggestion
- The entire mempool is kept in-memory, it is not just a cached subset of a larger structure.
```

**Review by stickies-v — COMMENTED** [review]
Thanks @LarryRuane, a few more styling nits but without having review the PR - not much more to add.

**Review by stickies-v — APPROVED** [review]
Thanks @LarryRuane @jonatack

**Pull request merged**